src/FixedDataTableBufferedRows.js

```diff
@@ -142,7 +144,7 @@ var FixedDataTableBufferedRows = React.createClass({
 
       this._staticRowArray[i] =
         <FixedDataTableRow
-          key={i}
+          key={rowKeyGetter(rowIndex)}
```

Comment:
Do you want me to make the change to the rowKeyGetter? Happy to do it if that's blocking a merge on this. Thanks!